subscribed_topics:
  dnnCV_estimator: estimate/drone_pose/dnnCV